workers/workers-v2/import-stock-order/import-stock-order-vend.js

```diff
@@ -320,14 +323,15 @@ function mapSpreadSheetDataToOrders(db, orderConfigModel, spreadSheetRows, userM
     let supplierStoreCodeFileHeader = _.findWhere(orderConfigModel.mappings.Store, {stockupHeader: 'supplierStoreId'}).fileHeader;
     let supplierStoreCodes = _.uniq(_.pluck(spreadSheetRows, supplierStoreCodeFileHeader));
     let skuFileHeader = _.findWhere(orderConfigModel.mappings.Product, {stockupHeader: 'sku'}).fileHeader;
+    let supplyPriceFileHeader = _.findWhere(orderConfigModel.mappings.Product, {stockupHeader: 'supplyprice'}).fileHeader;
     let orderQuantityFileHeader = _.findWhere(orderConfigModel.mappings.Inventory, {stockupHeader: 'orderQuantity'}).fileHeader;
     let fulfilledQuantityFileHeader = _.findWhere(orderConfigModel.mappings.Inventory, {stockupHeader: 'fulfilledQuantity'}).fileHeader;
     let skusToAdd = _.uniq(_.pluck(spreadSheetRows, skuFileHeader));
     //just to make sure to st\ringify if any sku contains just numbers
     let skusToAddStringified = _.map(skusToAdd, function (eachSku) {
         return typeof eachSku === 'string' ? eachSku : JSON.stringify(eachSku);
     });
-    let productModelInstances, storeModelInstances;
+    let supplypriceproductModelInstances, storeModelInstances;
```

Comment:
use camel case for variables

i thunk this variable is not being used

Reply:
Done ! there is no need to remove this variable